src/metric/definition.go

```diff
@@ -715,6 +715,16 @@ var KSMSpecs = definition.SpecGroups{
 			{Name: "isLimited", ValueFunc: prometheus.FromValue("kube_hpa_status_condition_limited")},
 		},
 	},
+	"node": {
+		TypeGenerator: kubeletMetric.FromRawGroupsEntityTypeGenerator,
```

Comment:
Isn't the `IDGenerator` needed here to identify each node? something like this 
```
IDGenerator:   prometheus.FromLabelValueEntityIDGenerator("kube_node_status_condition", "node"),
```

Reply:
Good point, I didn't notice that. However I just checked and the `node` section for the kubelet metrics do not have an ID generator either. I think this might be related to nodes not being complete "entities", i.e. they are not registered in the backend as such.

But I'd leave this open so we don't forget to doublecheck this.

Reply:
mmm interesting i made some test and is not needed, i will try to find where this magic is happen

Reply:
I think i [found it](https://github.com/newrelic/nri-kubernetes/blob/main/src/prometheus/definition.go#L211).

Reply:
Yeah, this is simply copied from https://github.com/newrelic/nri-kubernetes/blob/e34287a6a9e8f06cf883b57a58ca501bd862ab6b/src/metric/definition.go#L915-L916